drag_and_drop_v2/public/css/drag_and_drop_edit.css

```diff
@@ -141,9 +141,12 @@
     width: 18%;
 }
 
-.xblock--drag-and-drop--editor .zones-form .zone-row .title {
+.xblock--drag-and-drop--editor .zones-form .zone-row .title,
+.xblock--drag-and-drop--editor .zones-form .zone-row .description {
```

Comment:
Can you simplify this to `.xblock--drag-and-drop--editor .zones-form .zone-row input`, or would that have unwanted effects?